.github/workflows/terraform-orchestrator.yml

```diff
@@ -0,0 +1,146 @@
+name: Terraform
+
+on:
+  # Run plans for any PR base branch (not only main)
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
Je me demande si en specifiant ca tu n'exclues pas potentiellement des scenarios la ou ne pas le specifier englobe tous les types par defaut. Sauf si c'est voulu, j'enleverais la liste

Reply:
C'est voulu, sauf erreur on ne souhaite pas relancer de plan pour les autres types d'activité : https://docs.github.com/fr/actions/reference/workflows-and-actions/events-that-trigger-workflows#pull_request